src/style.css

```diff
@@ -0,0 +1,62 @@
+body {
+  margin: 0;
+}
+
+.header {
+  height: 60px;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.navbar {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.navbar-item {
+  list-style: none;
+  margin-right: 20px;
+}
+
+.navbar .navbar-item:last-child {
```

Comment:
Try to avoid increasing the specificity of the selector. One class is enough to identify the elements on the page. No need to write something like .block.colored-block. Use .block and .block--colored instead, and apply each of them to your element